charts/shoot-addons/charts/cluster-autoscaler/templates/deployment.yaml

```diff
@@ -25,6 +25,7 @@ spec:
       annotations:
 {{ toYaml .Values.podAnnotations | indent 8 }}
     {{- end }}
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
What if `.Values.podAnnotations` is empty?

Reply:
then it will be 

```
annotations:
  scheduler.alpha.kubernetes.io/critical-pod: ''
```

In all those examples I've put in the end, so we are sure that it's set

Reply:
But `annotations` is part of the `if` block and won't be there in case `.Values.podAnnotations` is empty, is it?